rust/operator-binary/src/controller.rs

```diff
@@ -407,7 +435,8 @@ fn build_server_rolegroup_daemonset(
         .collect::<Vec<_>>();
     let container_opa = ContainerBuilder::new("opa")
         .expect("invalid hard-coded container name")
-        .image(image)
+        .image(resolved_product_image.image.clone())
+        .image_pull_policy(resolved_product_image.image_pull_policy.clone())
```

Comment:
There should be a helper function like `image_from_product_image_selection` or similar

Reply:
Thanks. Done https://github.com/stackabletech/opa-operator/pull/385/commits/5e46eae030efdaf851953bbad6e93c2137f39240